core/src/main/scala/com/softwaremill/react/kafka/ReactiveKafka.scala

```diff
@@ -95,15 +96,67 @@ class ReactiveKafka {
     Sink.fromGraph(new KafkaGraphStageSink(producer))
   }
 
-  def graphStageSource[K, V](props: ConsumerProperties[K, V]) = {
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V]
+  ) = {
     val consumer = ReactiveKafkaConsumer(props)
     Source.fromGraph(new KafkaGraphStageSource(consumer))
   }
 
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V],
+    topicsAndPartitions: List[TopicPartition]
+  ) = {
+    val consumer = ReactiveKafkaConsumer(props, topicsAndPartitions)
+    Source.fromGraph(new KafkaGraphStageSource(consumer))
+  }
+
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V],
+    topicsAndPartitions: List[TopicPartition],
+    topicPartitionOffsetsMap: Map[TopicPartition, Long]
```

Comment:
There should be two functions, my mistake.  I am making the correction.  By the way, where ReactiveKafka functions use a List parameter, should that be a Set instead (since we want unique topic+partition[+offset] values)?